packages/plugin-core/src/commands/TogglePreview.ts

```diff
@@ -46,9 +55,16 @@ export class ShowPreviewCommand extends InputArgCommand<
    * be shown in preview. If unspecified, then preview will follow default
    * behavior of showing the contents of the currently in-focus Dendron note.
    */
-  async execute(opts?: ShowPreviewCommandOpts) {
+  async execute(opts?: TogglePreviewCommandOpts) {
     let note: NoteProps | undefined;
 
+    // If this class is used as TogglePreview (not as ShowCommand),
+    // hide (dispose) the previwe panel when it's already visible
+    if (!this._isShowCommand && this._panel.isVisible()) {
+      this._panel.hide();
+      return undefined;
+    }
+
```

Comment:
if the user is using `ShowPreview`, can we show a deprecation message and track it? see [[Deprecate Command|dendron://dendron.docs/pkg.plugin-core.dev.cook.deprecate-command]]